### PR TITLE
Add esy.json

### PR DESCRIPTION
**Pull request by andreypopp**
This adds esy.json so it can be developed/installed with `esy`:
```
% git clone https://github.com/aantron/repromise_lwt
% cd repromise_lwt
% esy i
% esy b
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
-_build/
```
Comment (aantron) [inline .gitignore:1]: Not that this matters, but does your editor remove the `/` somehow?

**Comment by aantron** [comment]
Thanks!

File: .gitignore (hunk 1 above)
Comment (andreypopp) [reply .gitignore:1]: esy creates `_build` symlink instead of `_build` dir, removing `/` was needed to make `git` ignore it.

Comment (aantron) [reply .gitignore:1]: Ah, thanks.

**Pull request merged**